.github/workflows/ppa-automation.yaml

```diff
@@ -26,6 +28,9 @@ jobs:
           if [[ "$GITREF" =~ ^refs/heads/releases/([0-9][^/]*) ]]; then
             BUILDREV="~rc$(git rev-list --count --first-parent origin/main..HEAD)"
             VERSION=$(echo $GITREF | cut -d'/' -f4)
+          elif [[ "$GITREF" =~ ^refs/tags/v([0-9a-z.]+) ]]; then
```

Comment:
should this match the tag regex?

Reply:
I suppose I was just being lazy, since we should only get into this path when the on-push hook is run anyways, and we're using the regex here to pick the version number out of the tag.